benchmark/single-source/ObjectiveCBridging.swift

```diff
@@ -709,3 +712,35 @@ public func run_ObjectiveCBridgeFromNSDateComponents(_ N: Int) {
   }
   #endif
 }
+
+var stringFromFile:String? = nil
+public func setup_stringFromFile() {
+  #if _runtime(_ObjC)
+  let url:URL
+  if #available(OSX 10.12, iOS 10.0, *) {
+    url = FileManager.default.temporaryDirectory.appendingPathComponent(
+      "sphinx.txt"
+    )
+  } else {
+    url = URL(fileURLWithPath: "/tmp/sphinx.txt")
```

Comment:
Does Foundation have a mktemp equivalent?

Reply:
The call in the other else branch is the best way, there's a clunkier version I could use on old OSs if it's a problem